book_homework5/book_homework5/Book.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace book_homework5
+{
+    class Book
+    {
+       string Title;
```

Comment:
What about auto[ properties](https://metanit.com/sharp/tutorial/3.4.php) instead of create get method for each property?